mne/viz/epochs.py

```diff
@@ -909,7 +909,7 @@ def _plot_update_epochs_proj(params, bools=None):
     params['projector'], _ = setup_proj(params['info'], add_eeg_ref=False,
                                         verbose=False)
 
-    start = params['t_start'] / len(params['epochs'].times)
+    start = int(params['t_start'] / len(params['epochs'].times))
```

Comment:
@jaeilepp can you verify this was correct? It's used to create a slice into `Epochs` so I think it's supposed to be an integer, but I'm not sure whether you intended truncation vs rounding here. (I was getting warnings about slices having non-integer values.)